### PR TITLE
Tic Tac Toe HW

### PR DESCRIPTION
**Pull request by ghost**
Completeness: 3
Comfort: 2

I feel comfortable with the HTML and CSS parts of this assignment and was able to complete those aspects of the assignment. I struggled with the JavaScript part of the assignment.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+window.onload = start;
+var boxes = document.getElementsByClassName("square");
+var moveTracker = 1;
+var winTracker = 0;
+var xMoves = [];
+var oMoves = [];
+
+//Set up event listener to listen for click
+
+for (var 1 = 0; i < boxes.length; i++) {
```
Comment (RobertAKARobin) [inline script.js:15]: Check your console! It throws this error: `Uncaught SyntaxError: Unexpected number` You have `var 1 = 0`. You can't use `1` as the name for a variable. Right now Javascript thinks you're trying to tell it that 1 is equal to 0, which is certainly isn't!

File: script.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  boxes[i].addEventListner('click', 'recordPlayerMove');
+}
+
+//define square combos that result in a win
+
+var winningCombos: [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6] ];
```
Comment (RobertAKARobin) [inline script.js:21]: Good strategy!

**Comment by RobertAKARobin** [comment]
In the future, try to code one line, then save and refresh, then another line, then save and refresh, and so on. This way you can take care of errors as soon as they pop up instead of having to go back and rewrite chunks of code.

**Comment by ghost** [comment]
Thanks for the feedback, Robin!

**Pull request closed without merging**